sway/config.c

```diff
@@ -300,6 +326,9 @@ static void config_defaults(struct sway_config *config) {
 	if (!(config->feature_policies = create_list())) goto cleanup;
 	if (!(config->ipc_policies = create_list())) goto cleanup;
 
+	// The keysym to keycode translation
+	config->keysym_translation = new_keysym_translation_data(getenv("XKB_DEFAULT_LAYOUT"));
```

Comment:
We generally call functions that create things with the `_create` suffix, and those which destroy things with the `_destroy` suffix.